web/cats/static/cats/cat_profile_style.css

```diff
@@ -1,17 +1,54 @@
+*{
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+/*Style for a header*/
+header {
+  position: fixed;
```

Comment:
I'd recommend to try `position: sticky` here. Check the [docs for `position: fixed` and `position: sticky`](https://developer.mozilla.org/en-US/docs/Web/CSS/position#fixed) and experiment a bit to see the difference between them.